src/components/button/index.js

```diff
@@ -9,18 +9,16 @@ const ButtonTheme = {
   SECONDARY: 'secondary'
 }
 
-const Button = ({ icon, children, onClick, type, disabled, theme }) => {
+const Button = ({ icon, children, onClick, type, disabled, theme, className }) => {
   return (
     <button
       onClick={onClick}
       disabled={disabled}
       type={type}
-      className={classnames(styles['button-content'], styles[theme])}
+      className={classnames(styles['button-content'], styles[theme], className)}
     >
       {icon && (
-        <svg className={styles['button-icon']} viewBox={icon.viewBox}>
-          <use xlinkHref={`#${icon.id}`}/>
-        </svg>
+        <img src={icon} className={styles['button-icon']} alt="Icon"/>
```

Comment:
as children is not required, you might have a button with no label. You should add aria-label prop here